javascript/javascript1/week3/index.js

```diff
@@ -0,0 +1,273 @@
+//FREECODECAMP EXERCISES
+//profile name - natsgt
+
+//ITEM ARRAY REMOVAL
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+function removeName(name) {
+  for (let i = 0; i < names.length; i++) {
+    if (names[i] === name) {
+      names.splice(i, 1);
+    }
+  }
+}
+
+removeName(nameToRemove);
+
+// Code done
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+//WHEN WILL WE BE THERE
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function calcTime(obj) {
+  let time = obj.destinationDistance / obj.speed;
+  let hour = Math.floor(time);
+  let minutes = Math.floor((time - Math.floor(time)) * 60);
+  let timeFormat = hour + (hour <= 1 ? " hour and " : " hours and ") + minutes + (minutes <= 1 ? " minute." : " minutes.");
```

Comment:
You can move "and" out of the ternary statement.
```suggestion
  let timeFormat = hour + (hour <= 1 ? " hour" : " hours") + " and " + minutes + (minutes <= 1 ? " minute." : " minutes.");
```

Reply:
Thanks, I'll change it.